lib/game_parser.rb

```diff
@@ -14,14 +14,11 @@ def get_game_info(games_path)
     contents.each do |row|
       game_id = row[:game_id]
       season = row[:season]
-      type = row[:type]
-      date_time = row[:date_time]
       away_team_id = row[:away_team_id]
       home_team_id = row[:home_team_id]
       away_goals = row[:away_goals]
       home_goals = row[:home_goals]
-      venue = row[:venue]
-      game = Game.new(game_id, season, type, date_time, away_team_id, home_team_id, away_goals, home_goals, venue)
+      game = Game.new(game_id, season, away_team_id, home_team_id, away_goals, home_goals)
```

Comment:
Thanks for removing the additional data points that were not necessary for the project!